src/patternfly/demos/DataList/data-list-expandable-compact-table.hbs

```diff
@@ -56,7 +56,7 @@
       {{#> table-td table-td--data-label="Action"}}
         <a href="#">Action link</a>
       {{/table-td}}
-      {{#> table-td table-td--action="true"}}
+      {{#> table-td table-td--IsAction=true}}
         {{> dropdown dropdown--id=(concat card--id "-dropdown-kebab-1") dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--IsPlain="true"}}
```

Comment:
Similar to above for removing these dropdowns in this file (also lines 91, 122 and 153)